crates/autopilot/src/arguments.rs

```diff
@@ -44,6 +44,11 @@ pub struct Arguments {
     #[clap(long, env)]
     pub tracing_node_url: Option<Url>,
 
+    /// An Ethereum node URL that supports `eth_call`s with state overrides to
+    /// be used exclusively for trade simulations.
+    #[clap(long, env)]
+    pub simulation_node_url: Option<Url>,
```

Comment:
Could you please also add some info to the CLI help text for `--trade-simulator` such that it says which kind requires which additional argument?